test/compliment/t_core.clj

```diff
@@ -22,7 +28,19 @@
     (strip-tags (core/completions "compliment.core/co"))
     => (just ["compliment.core/completions"])
 
+    (strip-tags (core/completions "'compliment.core/co"))
+    => (just ["compliment.core/completions"])
```

Comment:
Hmm, from real-world usage (in cider + company-mode local installations), it seems that the actual completion should also include the prefix, else company-mode will not offer the completion.

Which makes sense: for `#'x/`, returning `[x/a, x/b]` would mean that the `#'` prefix is lost, which is not what one means.

Reply:
Fixed in the second commit.

It's working beautifully!

<img width="565" alt="Screen Shot 2022-03-07 at 6 13 50 AM" src="https://user-images.githubusercontent.com/1162994/156972352-45d2464b-b0c1-4073-92d0-e223153391df.png">